modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts

```diff
@@ -143,6 +145,31 @@ function getPropertyDataFromBinarySource(
   return dataArray;
 }
 
+/**
+ * Parse featureTable.property.stringOffsetBufferView
```

Comment:
Again a bit more info about what that contains?

Reply:
Done